storybook/pages/AdvancedSettingsPage.qml

```diff
@@ -0,0 +1,267 @@
+        }
+    }
+    SplitView {
+        orientation: Qt.Vertical
+        SplitView.fillWidth: true
+
+        AdvancedView {
+            SplitView.fillWidth: true
+            SplitView.fillHeight: true
+            contentWidth: parent.width
+
+            accountSettings: mockData.accountSettings
+
+            advancedStore: AdvancedStore {
+                currentChainId: mockData.currentChainId
+                fleet: mockData.fleet
+                bloomLevel: mockData.bloomLevel
+                wakuV2LightClientEnabled: mockData.wakuV2LightClientEnabled
+                isTelemetryEnabled: mockData.isTelemetryEnabled
+                isAutoMessageEnabled: mockData.isAutoMessageEnabled
+                isDebugEnabled: mockData.isDebugEnabled
+                isCommunityHistoryArchiveSupportEnabled: mockData.isCommunityHistoryArchiveSupportEnabled
+
+                // customNetworksModel: advancedModule? advancedModule.customNetworksModel : []
+                customNetworksModel: []
+
+                isWakuV2: true
+
+                function toggleExperimentalFeature(feature) {
+                    logs.logEvent("advancedStore::toggleExperimentalFeature", ["feature"], arguments)
+                    if (feature === "wallet") {
+                        mockData.accountSettings.isWalletEnabled = !mockData.accountSettings.isWalletEnabled
+                    }
+                    if (feature === "browser") {
+                        mockData.accountSettings.isBrowserEnabled = !mockData.accountSettings.isBrowserEnabled
+                    }
+                    if (feature === "communityHistoryArchiveSupport") {
+                        mockData.isCommunityHistoryArchiveSupportEnabled = !mockData.isCommunityHistoryArchiveSupportEnabled
+                    }
+                    if (feature === "nodeManagement") {
+                        mockData.accountSettings.nodeManagementEnabled = !mockData.accountSettings.nodeManagementEnabled
+                    }
+                    if (feature === "communityPermissions") {
+                        mockData.accountSettings.isCommunityPermissionsEnabled = !mockData.accountSettings.isCommunityPermissionsEnabled
+                    }
+                    if (feature === "discordImportTool") {
+                        mockData.accountSettings.isDiscordImportToolEnabled = !mockData.accountSettings.isDiscordImportToolEnabled
+                    }
+                }
+
+                function setWakuV2LightClientEnabled(mode) {
+                    logs.logEvent("advancedStore::setWakuV2LightClientEnabled", ["mode"], arguments)
+                    mockData.wakuV2LightClientEnabled = mode
+                }
+                function setBloomLevel(mode) {
+                    logs.logEvent("advancedStore::setBloomLevel", ["mode"], arguments)
+                    mockData.bloomLevel = mode
+                }
+                function toggleDebug() {
```

Comment:
Something is wrong when trying to switch `debug` option:
"QML ConfirmationDialog: cannot find any window to open popup in."

Reply:
yes, it's an issue we have with (some) modals in storybook in general and will have to be fixed in some subsquent PR. It seems in part due to the way our modals work which is not consistent